packages/docs/community/release-process/smart-contracts.md

```diff
@@ -119,18 +131,21 @@ celocli governance:show --proposalID <proposalId> --jsonTransactions "upgrade_pr
 
 ### Verify Proposed Release
 
-Verify that the proposed upgrade activates contract addresses which match compiled bytecode from the tagged release candidate exactly. Include initialization data from the CGP to verify via (add `-i initialization_data.json` in that case)
+This script serves the same purpose as `verify-deployed` but for a not-yet
+accepted contract upgrade (in the form of the proposal.json you fetched in the step prior). It gives you the confidence that the branch specified in the `-b` flag in (same as `check-versions`) will be the resulting network state of the proposal if executed. It does so by going over all Celo Core Contracts and determining updates to the Registry pointers, proxy or implementation contracts and verifying their implied bytecode against the compiled source code.
+
+Additionally, include `initialization_data.json` from the CGP if any of the contracts have to be initialized.
 
 ```bash
 RELEASE_CANDIDATE="celo-core-contracts-v${N}.rc${X}"
 NETWORK=${"baklava"|"alfajores"|"mainnet"}
 # A -f boolean flag can be provided to use a forno full node to connect to the provided network
-yarn verify-release -p "upgrade_proposal.json" -b $RELEASE_CANDIDATE -n $NETWORK -f
+yarn verify-release -p "upgrade_proposal.json" -b $RELEASE_CANDIDATE -n $NETWORK -f -i initialization_data.json
 ```
 
 ### Verify Executed Release
 
-After a release executes via Governance, you can verify that the resulting network state reflects the tagged release candidate
+After a release executes via Governance, you can use `verify-deployed` again to check that the resulting network state reflects the tagged release candidate indeed:
```

Comment:
I like the added `indeed` flare 😄 . to me "indeed" flows better after a verb:  "checks that the resulting network `does indeed` reflect the tagged..." maybe it's a regional thing...